### ISSUE TITLE
COVID-19 API v1 

### ISSUE THREAD

**Issue by godwin-l** [postmanlabs/covid-19-apis#254]
Using this API you can get the world updates and India updates of COVID-19.

Base URL: https://covid19-api-godwin.herokuapp.com/

Documentation: https://medium.com/@alloygodwin1/coronavirus-covid-19-sars-cov-2-tracker-api-v1-built-to-track-global-corona-virus-outbreak-1c9d7ef84755

**Comment by kinlane** [comment]
Thank you for the submission. I am adding to our queue of work for creating a postman collection for your API, which will then allow us to add to the listing.

**Comment by godwin-l** [comment]
Hi @kinlane , Could you please tell me why the postman collection is not yet created for this API...

**Comment by kinlane** [comment]
Because nobody in the community has stepped up to create as of yet. Because we only have so many resources within Postman to invest in these efforts. Because we've all been working hard on many APIs and many collections during the pandemic, and we have to make hard choices about where we spent the precious  time and energy we have. I am sure there a couple other factors, but that should cover the base.

**Comment by godwin-l** [comment]
Shall I create the postman collection for this API and submit it over here... 

Then will it get added to list? @kinlane

**Comment by kinlane** [comment]
Yes. Once there is a collection added I switch the tag to add-to-api-list, and it gets added in the next build.
